### ISSUE TITLE
Can you share the pre-trained models?

### ISSUE THREAD

**Issue by EdVince** [kynthesis/StarSRGAN#3]

**Comment by Ivan29B** [comment]
It would not be bad to study the models themselves, because training a model from scratch is very resource-intensive

**Comment by kynthesis** [comment]
Apologies, I started this project a year ago, and I have been writing my paper since then. 
It has been quite a mess for me with numerous attempts at trying, testing, debugging, and saving models. 
All I have left are some old models (StarSRGAN v0.4) and benchmark sheets. 
However, the model architecture and config files still worked well.
Currently, I am also researching another project called StarSRGAN 2.0. Thank you for giving StarSRGAN 1.0 a try!

**Comment by Phhofm** [comment]
Ah dang thats too bad about the pretrains, would have been great to have them.

I can understand the training process being messy, especially since yall trained like 20 different models (like with dual-preceptuall loss or not, multi-disc or not etc) which is time and ressource intensive, I was impressed, thanks for trying these out and providing metrics and visual results (this lets me for example look at the visual outputs and think for myself that while v3 nearly retains the same details as v5, it has less training complexity, meaning i am not so sure if its worth enabling multi-disc and dual-loss in the config for training). 

Keep us informed about your progress on StarSRGAN2.0 ;) 

(maybe one suggestion I would have, currently the x4 scale seems hardcoded into both inference&training, but I think especially the lite model could profit from the ability of making a x2 model instead (x4 simply feels a bit much for lite currently results wise from my testing/training a bit, thats just my feeling, id rather do a x2 model on litesrnet. 

And also if you were to do different versions again, would be great to have example configs of these version, like with dual-loss enabled and disabled. In the current configs like train_starsrgan.yml in the options folder, it uses PerceptualLoss and not dual I believe? (would I simply rename this loss to DualPerceptualLoss and it would work? Or would I need to specify resnet somewhere in the config? not sure, more example configs could help with such things if there are different versions, so I could simply train a V3 version like in the paper). 
(sorry for long text)

**Comment by kynthesis** [comment]
You have a sharp eye, Philip.

StarSRGAN V3 is the most stable variation according to its result on different hyperparameter settings. Generally, I suggest you stay with it for saving time and resources. Means that you can ignore the DualPerceptualLoss and MultiscaleDiscriminator.

StarSRGAN can be trained on x2 scale, but I didn't mention in my paper cause the conference wanted to limit its length down from 15 pages to 10 pages. I haven't tested on x2 settings, you may be the first one do that with StarSRGAN.

**Comment by cbroker1** [comment]
@kynthesis where can we find StarSRGAN V3?

**Comment by xduzhangjiayu** [comment]
@kynthesis 
Hi, thanks for the awesome work, have the StarSRGAN 2.0 already released? where can I find it? 
Thanks
